osu.Game/Skinning/SkinManager.cs

```diff
@@ -49,24 +49,26 @@ public class SkinManager : ModelManager<SkinInfo>, ISkinSource, IStorageResource
 
         public readonly Bindable<Skin> CurrentSkin = new Bindable<Skin>();
 
-        public readonly Bindable<Live<SkinInfo>> CurrentSkinInfo = new Bindable<Live<SkinInfo>>(Skinning.DefaultSkin.CreateInfo().ToLiveUnmanaged())
-        {
-            Default = Skinning.DefaultSkin.CreateInfo().ToLiveUnmanaged()
-        };
+        public readonly Bindable<Live<SkinInfo>> CurrentSkinInfo = new Bindable<Live<SkinInfo>>(ArgonSkin.CreateInfo().ToLiveUnmanaged());
 
         private readonly SkinImporter skinImporter;
 
         private readonly IResourceStore<byte[]> userFiles;
 
         /// <summary>
-        /// The default skin.
+        /// The default "triangles" skin.
```

Comment:
```suggestion
        /// The default "argon" skin.
```

Reply:
I've just removed this as it's a private now.